api/src/pcapi/core/providers/api.py

```diff
@@ -45,6 +45,7 @@ class T_UNCHANGED(enum.Enum):
 def create_venue_provider(
     provider_id: int,
     venue_id: int,
+    author: users_models.User,
```

Comment:
```suggestion
    current_user: users_models.User,
```
le rennomer en current_user permetrait d'être plus clair parce que ce n'est pas forcement le author qu'on lui passe.